client/web/compose/src/components/PageBlocks/RecordListBase.vue

```diff
@@ -155,22 +156,36 @@
           <div
             class="d-flex align-items-baseline my-auto pt-1 text-nowrap h-100"
           >
-            {{ $t('recordList.selected', { count: selected.length, total: items.length }) }}
-            <b-button
-              variant="link"
-              class="p-0 text-decoration-none"
-              @click.prevent="handleSelectAllOnPage({ isChecked: false })"
-            >
-              ({{ $t('recordList.cancelSelection') }})
-            </b-button>
+            <div>
+              {{ selectedAllRecords ? $t('recordList.selectedFromAllPages', { count: getPagination.count }) : $t('recordList.selected', { count: selected.length, total: items.length }) }}
+              <b-button
```

Comment:
You can remove the cancel button since it's a bit confusing now with Select on all pages button.